docs/features/compilation-from-portable-pdb.md

```diff
@@ -0,0 +1,90 @@
+## compilation-from-portable-pdb
+
+Compilation from portable PDBs today is not completely possible, but is desirable in order to help reconstruct a compilation from source provided via source link or embedded in a pdb. Motivation is derived from [roslyn 41395](https://github.com/dotnet/roslyn/issues/41395).
+
+This document is restricted to the following assumptions:
+
+1. The benefit is for builds with `-deterministic` and published to the symbol server.
+2. Source generator and analyzer references are not needed for this task. They may be useful, but are out of scope for this document.
+3. Any storage capacity used for PDBs and source should not impact this feature, such as compression algorithm.
+4. Only Portable PDB files will be included for this spec. We can expand this feature past these once it is implemented and proven needed elsewhere.
```

Comment:
This will break round tripping between native and portable PDBs. Don't our own tools depend on this being possible?

Reply:
We don't depend on it. The tests that compare PDB formats already account for other differences between the formats.

Reply:
Marking as resolved so I can keep track of open concerns. Feel free to reopen if it's not.